### PR TITLE
feat(backup): support for advancedBackupSettings

### LINKED ISSUE #14803: (backup): Support for advancedBackupSettings
AWS Backup BackupPlan construct is missing support for AdvancedBackupSetting
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-backup-backupplan-advancedbackupsettingresourcetype.html

### Use Case

This is needed to support Windows VSS in AWS Backup configured with CDK

### Proposed Solution

Add something like:  `backupOptions: {"WindowsVSS": "enabled"}` to support the advanced backup options

### Other

* [ ] :wave: I may be able to implement this feature request
* [ ] :warning: This feature might incur a breaking change

---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by Patil2099**
Fixes #14803

### CODE REVIEW

**Comment by Patil2099** [comment]
Can you please review this @MrArnoldPalmer ?

File: packages/@aws-cdk/aws-backup/lib/plan.ts [hunk 1]
```diff
@@ -125,6 +126,7 @@ export class BackupPlan extends Resource implements IBackupPlan {
     const plan = new CfnBackupPlan(this, 'Resource', {
       backupPlan: {
         backupPlanName: props.backupPlanName || id,
+        backupOptions: props.backupOptions || {},
```
Comment (BenChaimberg) [inline packages/@aws-cdk/aws-backup/lib/plan.ts:129]: This is not the correct location for this value, please consult [the documentation for this resource](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-backup.CfnBackupPlan.html).

**Review by BenChaimberg — CHANGES_REQUESTED** [review]
Please consult [the contributing guide](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md) for the requirements for pull requests in this repository.

In particular, please address the following issues:
- PRs should be named following [the conventional commit specification](https://www.conventionalcommits.org/en/v1.0.0/) (I fixed this for you already)
- public interfaces must have associated documentation
- new features must add to the README
- changes should modify or create tests
- build must pass

Besides these (somewhat) administrative issues, I'm not sure if this is the correct solution to implement #14803. Given that the only supported key for BackupOptions is "WindowsVSS", adding a property like "windowsVssBackup?: boolean" and generating the correct options will be more literate to the user.

**Comment by TheRealAmazonKendra** [comment]
This PR has been deemed to be abandoned, and will be closed. Please create a new PR for these changes if you think this decision has been made in error.

**Pull request closed without merging**